src/exchanges/gemini-client.js

```diff
@@ -255,6 +255,18 @@ class GeminiClient extends EventEmitter {
 
     if (!market) return;
 
+    if (msg.type === "heartbeat") {
```

Comment:
So is the `heartbeat` event always sent when you have a socket active?  If so, maybe add an additional comment that the heartbeat event is automatically emitted for live sockets.  Thank you for adding the existing comments!

Reply:
yes I added some bullet points copy/pasted from Gemini docs